### ISSUE TITLE
[ENG-412] Labelbox2coco cannot handle skipped labels or labels without any objects

### ISSUE THREAD

**Issue by kobbled** [Labelbox/Labelbox#101]
Labelbox2coco script errored trying to process these two types of labels from my export.json file. 
- images that were skipped
"Label":"Skip"
- images where no objects where found
"Label":{"model":"none"}

**Comment by manuaero** [comment]
Thanks -- we will be updating this soon.

**Comment by feynmanliang** [comment]
Thank you for the bug report. We will be addressing this shortly after #109

**Comment by feynmanliang** [comment]
Hi @kobbled , I've added a test to verify that your first issue wrt "Label: Skip" is resolved.

I'm not clear how you're able to get the second case. If it is still a problem, can you open another issue with repro steps leading to how you got a `{"model": "none"}` label? thanks!
